### PR TITLE
BluetoothUUID wpt tests

### PR DESCRIPTION
**Pull request by zakorgy**
Wpt tests for the `BluetoothUUID` functions.

---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [x] There are tests for these changes

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/14207)

### CODE REVIEW

File: tests/wpt/mozilla/tests/mozilla/bluetooth/idl-BluetoothUUID.html [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+<script src="/resources/testharnessreport.js"></script>
+<script src="/_mozilla/mozilla/bluetooth/bluetooth-helpers.js"></script>
+<script>
+'use strict';
+
+test(() => {
+  assert_equals(BluetoothUUID.getService(base_alias), base_uuid);
+  assert_equals(BluetoothUUID.getCharacteristic(base_alias), base_uuid);
+  assert_equals(BluetoothUUID.getDescriptor(base_alias), base_uuid);
+}, '0x0 should produce valid UUID.');
```
Comment (jdm) [inline tests/wpt/mozilla/tests/mozilla/bluetooth/idl-BluetoothUUID.html]: This description doesn't match what's being tested.

**Review by jdm — CHANGES_REQUESTED** [review]
Looks good apart from one comment!

**Comment by zakorgy** [comment]
I changed the description in the first test function, also moved the used variables in the test file, because these are only used in this test.

**Comment by jdm** [comment]
Even better!
@bors-servo: r+

**Pull request merged**